src/index.html

```diff
@@ -9,10 +9,72 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The href attribute should link to a CSS file, not a SCSS file. Please change 'styles/main.scss' to 'styles/main.css'.